### PR TITLE
fix(sms): remaining chars estimate

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | internal DTRSD-32868
| License          | BSD 3-Clause

## Description

fix remaining chars in sms composition not correctly displayed

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html [hunk 1]
```diff
@@ -80,8 +80,7 @@ <h2 class="heading_underline" data-translate="sms_message_title"></h2>
                 <oui-checkbox
                     name="noCommercialClause"
                     id="noCommercialClause"
-                    model="$ctrl.model.noStopClause"
-                    on-change="$ctrl.getMessageDetails(modelValue)"
```
Comment (JeremyDec) [inline packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html:84]: No more calculation made after checkbox changed ?

File: packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.controller.js [hunk 2]
```diff
@@ -12,16 +13,20 @@ export default class SmsCompositionController {
...
-      SmsCompositionController.isShortNumber(this.model.sender)
-        ? true
-        : this.model.noStopClause;
-    const suffix = noStopValue !== undefined ? noStopValue : isShort;
-    this.model.messageDetails = this.TucSmsMediator.getSmsInfoText(
-      this.model.message,
-      !suffix,
-    );
+    // timeout is here because the onChange event of the oui-textarea
+    // is triggered before the model is updated
```
Comment (marie-j) [inline packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.controller.js:17]: Isn't the new model the value passed as parameter ?

**Review by marie-j — APPROVED** [review]

File: packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html (hunk 1 above)
Comment (frenautvh) [reply packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html:84]: from what i understand only the noStopClause does change the calculation, am i missing something ?

Comment (JeremyDec) [reply packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html:84]: Oh I didn't saw you change from `noStopClause` to `noCommercialClause`, my bad, it's ok then

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.controller.js (hunk 2 above)
Comment (frenautvh) [reply packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.controller.js:17]: yes you're right, the issue here is that this method is also used as a callback for the on-change method on some radio-buttons in the template, so changing the method signature (textarea value as parameter) will conflict with the behavior of the radio buttons that are passing their model value as parameter

**Pull request merged**